cmd/config/set.go

```diff
@@ -195,12 +209,21 @@ func configSetContext(cmd *cobra.Command, args []string) {
 		if err != nil {
 			log.Fatal(err.Error())
 		}
+		if !flags.Changed("patch") {
+			URLClearConditions(ctxPtr)
+		}
 		ctxPtr.URL = cleanedUrl
 	}
 	if flags.Changed("tenant") {
+		if ctxPtr.AuthMethod != "jwt" && !flags.Changed("patch") {
```

Comment:
actually, the tenant is allowed for the JWT auth mode. Also the check above is for non-JWT mode... Can you please check/re-consider here?